src/istio/mixerclient/check_cache.cc

```diff
@@ -147,18 +148,23 @@ Status CheckCache::CacheResponse(const Attributes &attributes,
   }
   utils::HashType signature;
   if (!referenced.Signature(attributes, "", &signature)) {
-    GOOGLE_LOG(ERROR) << "Response referenced mismatchs with request";
-    GOOGLE_LOG(ERROR) << "Request attributes: " << attributes.DebugString();
-    GOOGLE_LOG(ERROR) << "Referenced attributes: " << referenced.DebugString();
+    if (MIXER_ERROR_ENABLED) {
+      MIXER_ERROR(
+          "Response referenced mismatchs with request.  Request attributes: "
```

Comment:
s/mismatchs/mismatched

Also, how frequently do we expect this to trigger?  This will push a bunch of stuff to the log.

Reply:
I didn't add that one, just converted it to use Envoy log formatting instead of using libprotobufs logger.   So the frequency is the same, but now it can be controlled with envoy's log threshold and format is consistent

Reply:
(will fix the typo though)

Reply:
ok. yeah, i guess this should be rare. just curious.  we probably need a general audit of logging across Istio.